packages/protocol/test/governance/release_gold.ts

```diff
@@ -930,6 +1037,77 @@ contract('ReleaseGold', (accounts: string[]) => {
     })
   })
 
+  describe('#expire', () => {
+    beforeEach(async () => {
+      await createNewReleaseGoldInstance(releaseGoldDefaultSchedule, web3)
+    })
+
+    describe('when called before expiration time has passed', () => {
+      it('should revert', async () => {
+        await assertRevert(releaseGoldInstance.expire({ from: releaseOwner }))
+      })
+    })
+
+    describe('when the contract has finished releasing', () => {
+      beforeEach(async () => {
+        const timeToTravel = 12 * MONTH
+        await timeTravel(timeToTravel, web3)
+      })
+
+      it('should revert before `EXPIRATION_TIME` after release schedule end', async () => {
+        await assertRevert(releaseGoldInstance.expire({ from: releaseOwner }))
+      })
+
+      describe('when `EXPIRATION_TIME` has passed after release schedule completion', () => {
+        beforeEach(async () => {
+          const [releaseGoldStartTime, , , ,] = await releaseGoldInstance.releaseSchedule()
+          const expirationTime = await releaseGoldInstance.EXPIRATION_TIME()
+          const timeToTravel = releaseGoldStartTime.plus(expirationTime).toNumber()
+          await timeTravel(timeToTravel, web3)
+        })
+
+        describe('when not called by releaseOwner', () => {
+          it('should revert', async () => {
+            await assertRevert(releaseGoldInstance.expire())
+          })
+        })
+
+        describe('when called by releaseOwner', () => {
+          it('should succeed', async () => {
+            await releaseGoldInstance.expire({ from: releaseOwner })
+          })
+        })
+
+        describe('when an instance is expired', () => {
+          beforeEach(async () => {
+            await releaseGoldInstance.expire({ from: releaseOwner })
+          })
+
+          it('should revoke the contract', async () => {
+            const isRevoked = await releaseGoldInstance.isRevoked()
+            assert.equal(isRevoked, true)
+          })
+
+          it('should set the released balance at revocation to total withdrawn', async () => {
+            const [, releasedBalanceAtRevoke] = await releaseGoldInstance.revocationInfo()
+            // 0 gold withdrawn at this point
+            assertEqualBN(releasedBalanceAtRevoke, 0)
+          })
+
+          it('should allow refund of all remaining gold', async () => {
```

Comment:
Can you check if there's a test to ensure a non-releaseOwner can't withdraw after the revocation?

Reply:
There is a test that a beneficiary cannot withdraw more than their released gold at revoke time, which covers this logic, but potentially I can add a test later of a totally associated address, but there's no reason that wouldn't work.